src/panels/templates/AutomaticCreateCluster.json

```diff
@@ -158,21 +158,10 @@
             "name": "[parameters('resourceName')]",
             "dependsOn": [],
             "properties": {
-                "enableRBAC": "[parameters('enableRBAC')]",
-                "nodeResourceGroup": "[parameters('nodeResourceGroup')]",
-                "nodeResourceGroupProfile": "[parameters('nodeResourceGroupProfile')]",
```

Comment:
we need to remove parameters for this as well and test it (some more things to be done) @sabbour I can do a PR against your fork so that it will be collaborative PR.